vector/src/main/java/im/vector/app/features/call/dialpad/DialPadLookup.kt

```diff
@@ -42,18 +42,23 @@ class DialPadLookup @Inject constructor(
         val sipUserId = thirdPartyUser.userId
         val nativeLookupResults = session.sipNativeLookup(thirdPartyUser.userId)
         // If I have a native user I check for an existing native room with him...
-        val roomId = if (nativeLookupResults.isNotEmpty()) {
+        if (nativeLookupResults.isNotEmpty()) {
             val nativeUserId = nativeLookupResults.first().userId
             if (nativeUserId == session.myUserId) {
                 throw Failure.NumberIsYours
             }
-            session.roomService().getExistingDirectRoomWithUser(nativeUserId)
-            // if there is not, just create a DM with the sip user
-                    ?: directRoomHelper.ensureDMExists(sipUserId)
-        } else {
-            // do the same if there is no corresponding native user.
-            directRoomHelper.ensureDMExists(sipUserId)
+            var nativeRoomId = session.getExistingDirectRoomWithUser(nativeUserId)
```

Comment:
You are maybe not using the latest develop from mainline?
The code should be session **.roomService()** .getExistingDirectRoomWithUser(nativeUserId) now

Reply:
oops, completely missed that - copy pasted that part of the change from our fork, sorry.